spec/linter-eslint-spec.js

```diff
@@ -50,13 +50,12 @@ function copyFileToDir(fileToCopyPath, destinationDir) {
  */
 // eslint-disable-next-line import/prefer-default-export
 export function copyFileToTempDir(fileToCopyPath) {
-  return new Promise(async (resolve) => {
-    const tempFixtureDir = fs.mkdtempSync(tmpdir() + path.sep)
-    resolve(await copyFileToDir(fileToCopyPath, tempFixtureDir))
-  })
+  const tempFixtureDir = fs.mkdtempSync(tmpdir() + path.sep)
+  // copyFileToDir returns a promise
```

Comment:
🔥 this comment 😉.

Reply:
Happy to nuke the comment if you want. I added this comment for exactly the reason you are concerned about this change, a little reminder that it's async underneath. A simple comment is much less overhead and much less confusing than an  a `Promise` wrapping an `async function` so we can `await` a `Promise` just to turn it back into a `Promise` again. 

Perhaps just marking the top level `copyFileToTempDir()` as `async` even though we won't  use it (along with nuking the comment) would be a nice comprimise?

Reply:
done